examples/go/README.md

```diff
@@ -0,0 +1,15 @@
+## Import vml package into your project
+### Initialize go.mod file and download the package
+```
+go mod init examople.com/example
```

Comment:
Gotta love "examople"... Please, let's leave it like this! 😄

Reply:
It's up to your approval :D